nodejs/awsx/aws_test.go

```diff
@@ -62,9 +63,15 @@ func Test_Examples(t *testing.T) {
 	}
 
 	shortTests := []integration.ProgramTestOptions{
-		testBase.With(integration.ProgramTestOptions{
-			Dir: path.Join(cwd, "../examples/cluster"),
-		}),
+		integration.ProgramTestOptions{
+			Config: map[string]string{"aws:region": envRegion},
+			Dir:    path.Join(cwd, "../examples/cluster"),
+			Dependencies: []string{
+				"@pulumi/awsx",
+			},
+			Quick:       true,
+			SkipRefresh: true,
+		},
 		testBase.With(integration.ProgramTestOptions{
```

Comment:
The mixing of tests based on `testBase` with others seems like it could be fragile in the future.  Should we rename or restructure a little to make it clearer what tests are testing what (and how new tests should be added)?